### ISSUE TITLE
Issue with 'stacklevel'

### ISSUE THREAD

**Issue by aclong** [HTenkanen/transx2gtfs#18]
I got this error message:

```

multiprocessing.pool.RemoteTraceback:
"""
Traceback (most recent call last):
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/site-packages/transx2gtfs/converter.py", line 172, in process_files
    calendar_dates = get_calendar_dates(gtfs_info)
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/site-packages/transx2gtfs/calendar_dates.py", line 83, in get_calendar_dates
    stacklevel=2)
TypeError: Argument given by name ('stacklevel') and position (3)
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/site-packages/transx2gtfs/converter.py", line 258, in convert
    pool.map(process_files, workers)
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/multiprocessing/pool.py", line 288, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/mydir/.conda/envs/transx-converter/lib/python3.6/multiprocessing/pool.py", line 670, in get
    raise self._value
TypeError: Argument given by name ('stacklevel') and position (3)
>>>


```

I'll check it tomorrow but thought I'd post it here

**Comment by HTenkanen** [comment]
@aclong Cool, thanks for reporting! I'll check this.

**Comment by HTenkanen** [comment]
@aclong Fixed in #20.
